frontend/src/app/modules/home/home-page/home-page.component.ts

```diff
@@ -19,4 +27,13 @@ export class HomeComponent implements OnInit, OnDestroy {
     ngOnDestroy() {
         this.broadcastHub.stop();
     }
+
+    addDashboard(dashboard: Dashboard) {
+        this.displayModal = false;
+        this.dashboardService.addDashboard(dashboard);
+    }
+
+    ngOnChanges(): void {
+        this.dashboards = this.dashboardService.getAll();
+    }
```

Comment:
Why do you need this?

Reply:
![5](https://user-images.githubusercontent.com/66973922/127771549-e1e34de1-dc78-430d-8a48-4f9f85ac2a23.png)
I use dashboards here.

Reply:
Remove.